lib/asciidoc-reference-check.js

```diff
@@ -308,6 +327,41 @@ function checkReferences(filePath, basePath) {
         }
       }
     }
+
+    //find external references that use the link:URI[some description] syntax
+    if (line.match(/link:[^\[]+\[[^\]]*\]/g)) {
+      //console.log('LINE-----',line)
+      let extractLink = line.match(/link:[^\[]+\[/g); //there may be more than one matching items
+      for (let i = 0; i < extractLink.length; i++) {
+        let newReference = extractLink[i];
+        newReference = newReference.replace('link:', '');
+        newReference = newReference.replace('[', '');
+
+        //distinguish local files from URLs
+        if (newReference.match(/^(http|https):\/\//g)) {
+          externalURLs.push(newReference);
```

Comment:
Before pushing should we filter out some valid hyperlinks which are used as example, such as `http://www.example.com`, `http://localhost:8080` etc. 

something like (not tested)
```
if (newReference.match(/^(http|https):\/\//g)) {
          if (
            !newReference.match(
              /(example\.(?:com|org|test)|localhost|my\.proxy|http(?:s|):\/\/\$|http(?:s|):\/\/\%|location_of_rpm_server|\d{1,3}.\d{1,3}.\d{1,3}.\d{1,3}|someaccount\.)/
            )
        )
          externalURLs.push(newReference);
...
```

Reply:
I created the regex from some OpenShift docs examples we use. There could be a better more generic one, i am not aware of.

Reply:
Hi!
Good idea.
For more flexibility, we could even allow the user (through of configuration Option) to define a set of URLs that should not be checked.
From my Point of view we could add this Feature later (as the URL check can already be disabled completely by a config Option, so that the user would not get any spurious Messages)

Note: the domain www.example.org/com already exist, so their is no Need for an exception in this case.